src/xp.runner/commands/Version.cs

```diff
@@ -10,6 +13,8 @@ public class Version : Command
         /// <summary>Command line arguments.</summary>
         protected override IEnumerable<string> ArgumentsFor(CommandLine cmd)
         {
+            var self = Assembly.GetExecutingAssembly();
+            Console.WriteLine("Runners {0} {{ .NET {1} }} @ {2}", self.GetName().Version, self.ImageRuntimeVersion, Paths.Binary());
```

Comment:
While this is cool because it works with any framework version writing specification tests becomes harder. Needs to be considered...